examples/metalk8s-solution-example/ui/src/ducks/app/clockServer.js

```diff
@@ -85,18 +85,18 @@ export function* createClockServer({ payload }) {
   const result = yield call(
     ApiK8s.createClockServer,
     body,
-    `${environment}-example-solution`
+    `${environment}-${SOLUTION_NAME}`
   );
   if (!result.error) {
-    yield call(fetchClockServer, `${environment}-example-solution`);
+    yield call(fetchClockServer, `${environment}-${SOLUTION_NAME}`);
     yield call(history.push, `/environments/${environment}`);
   }
 }
 
 export function* editClockServer({ payload }) {
   const { name, environment, timezone, version } = payload;
   const body = {
-    apiVersion: 'example-solution.metalk8s.scality.com/v1alpha1',
+    apiVersion: `${SOLUTION_NAME}.metalk8s.scality.com/v1alpha1`,
```

Comment:
Conceptually, this is not portable to other Solutions. A better approach would define (in `constants.js`):

```javascript
const SOLUTION_API_GROUP = `${SOLUTION_NAME}.metalk8s.scality.com`
```

And you could use this value in here, only specifying `v1alpha1` for each CR (it could differ between them, so no need to attach it in `constants.js`).